### PR TITLE
OEL-164: Drupal 9 compatibility fixes for oe_contact_forms.

### PR DESCRIPTION
**Pull request by escuriola**
### Description

Drupal 9 compatibility fixes

### Change log

- Added: core version 8 || 9, other additions
- Changed: "short" import classes, minor code refactor from phpcs report
- Deprecated:
- Removed: PHP 7.2
- Fixed: 
- Security: 

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Comment by escuriola** [comment]
phpunit tests passed 
behat tests passed

**Comment by GilNovacomm** [comment]
Hello @escuriola 
The branch naming is wrong the correct is:
git checkout -b [ticket] / git checkout -b OEL-164
When committing:
git commit "OEL-164: <message>."

File: tests/src/FunctionalJavascript/CorporateContactFormTest.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 namespace Drupal\Tests\oe_contact_forms\FunctionalJavascript;
 
+use Drupal;
```
Comment (drishu) [inline tests/src/FunctionalJavascript/CorporateContactFormTest.php:7]: don't do this anywhere, not in tests, not in src, not anywhere, the drupal standard is to use backslash when using native classes, see https://www.drupal.org/project/drupal/issues/2053489#comment-7863665

**Review by drishu — CHANGES_REQUESTED** [review]
also please amend the tests, ex assertEqual is deprecated, setUp: void, etc

File: tests/src/FunctionalJavascript/CorporateContactFormTest.php (hunk 1 above)
Comment (escuriola) [reply tests/src/FunctionalJavascript/CorporateContactFormTest.php:7]: Yep, sorry. We were checking drupal-rector with @GilNovacomm and it was not working (no output). Setting "short imports" feature to true from rector.yml make it work and made this changes automatically. Don't worry, I take note and never used it again.

File: oe_contact_forms.module [hunk 2]
```diff
@@ -268,9 +269,9 @@ function _oe_contact_forms_get_privacy_default(string $uri): string {
   elseif ($scheme === 'entity') {
     [$entity_type, $entity_id] = explode('/', substr($uri, 7), 2);
     // Show the 'entity:' URI as the entity autocomplete would.
-    // @todo Support entity types other than 'node'. Will be fixed in
-    //    https://www.drupal.org/node/2423093.
-    if ($entity_type == 'node' && $entity = \Drupal::entityTypeManager()->getStorage($entity_type)->load($entity_id)) {
+    // @todo   Support entity types other than 'node'. Will be fixed in
```
[the remark this answers is not in the document]
Comment (escuriola) [reply oe_contact_forms.module:272]: phpcs indicates an error but I think in the line below. I'm reverting it

**Pull request closed without merging**